blackbox/docs/administration/ssl.txt

```diff
@@ -179,6 +184,17 @@ information including the different SSL modes please visit the
 .. _psql documentation: https://www.postgresql.org/docs/current/static/app-psql.html
 
 
+Connecting to a CrateDB MQTT Broker With SSL/TLS
+------------------------------------------------
+A CrateDB node configured to run as :ref:`MQTT broker <ingest_mqtt>` can 
+accept encrypted connections if ssl ingestion is enabled, namely if the 
+``ssl.ingestion.mqtt.enabled`` configuration is set to ``true`` and the 
+configured port ``ingestion.mqtt.port`` is ``8883``.
+
+There are various MQTT client implementations available and  whichever you will 
+use, you will need to configure the Certificate Authorities that are to be 
+trusted by the client and the certificate that will be used for TLS based 
+authentication.   
```

Comment:
Not necessarily, the trust store is optional. Should we skip this paragraph and just link to the SSL configuration page?

Reply:
Hmm, this is the SSL configuration page, but you are right that this makes it sound mandatory.